src/main/java/org/folio/fqm/service/CrossTenantQueryService.java

```diff
@@ -115,4 +115,12 @@ private boolean ecsEnabled(Map<String, String> ecsTenantInfo) {
   public boolean ecsEnabled() {
     return ecsEnabled(getEcsTenantInfo());
   }
+
+  public boolean isCentralTenant() {
+    return executionContext.getTenantId().equals(getCentralTenantId());
+  }
```

Comment:
I'd follow the pattern that the other method here use, where there's overloaded with a `private` version that takes the `Map<String, String> ecsTenantInfo` parameter. Then, I'd update `getTenantsToQuery()` to use the `private` version, so that all of the "is this the central tenant?" logic runs through that one method